machine/dni/dni_3048up/kernel/platform-dni_3048up.patch

```diff
@@ -0,0 +1,1933 @@
++                                        reg = <0x4>;
++                                        sfp_eeprom@50 {
++                                                compatible = "at,24c04";
++                                                reg = <0x50>;
++                                                label = "port49";
++                                        };
++                                };
++                                i2c@5 {
++                                        #address-cells = <0x1>;
++                                        #size-cells = <0x0>;
++                                        reg = <0x5>;
++                                        sfp_eeprom@50 {
++                                                compatible = "at,24c04";
++                                                reg = <0x50>;
++                                                label = "port50";
++                                        };
++                                };
++                                i2c@6 {
++                                        #address-cells = <0x1>;
++                                        #size-cells = <0x0>;
++                                        reg = <0x6>;
++                                        sfp_eeprom@50 {
++                                                compatible = "at,24c04";
++                                                reg = <0x50>;
++                                                label = "port51";
++                                        };
++                                };
++                                i2c@7 {
++                                        #address-cells = <0x1>;
++                                        #size-cells = <0x0>;
++                                        reg = <0x7>;
++                                        sfp_eeprom@50 {
++                                                compatible = "at,24c04";
++                                                reg = <0x50>;
++                                                label = "port52";
++                                        };
++                                };
++                        };
++                };
++        };
++};
+\ No newline at end of file
+diff --git a/arch/arm/mach-iproc/Kconfig b/arch/arm/mach-iproc/Kconfig
+old mode 100644
+new mode 100755
+index ed24988..65fda7a
+--- a/arch/arm/mach-iproc/Kconfig
++++ b/arch/arm/mach-iproc/Kconfig
+@@ -41,6 +41,11 @@ config MACH_GH
+ 	help
+ 	  Support for the Broadcom Greyhound bring-up board.
+ 
++config MACH_DNI_3048UP
++	bool "Support DNI 3048UP board"
++	help
++	  Support for the DNI 3048UP board.
++
+ endchoice
+ 
+ config MACH_IPROC_P7
+diff --git a/arch/arm/mach-iproc/board_bu.c b/arch/arm/mach-iproc/board_bu.c
+old mode 100644
+new mode 100755
+index b45588d..0365ac6
+--- a/arch/arm/mach-iproc/board_bu.c
++++ b/arch/arm/mach-iproc/board_bu.c
+@@ -51,7 +51,7 @@
+ #endif /* HNDCTF */
+ 
+ #include <linux/amba/pl022.h>
+-
++#if 0
```

Comment:
Can you change to macro  i.e.(!defined (DNI_3048UP) 

Does this block cause any issue on DNI-3048UP board